src/lua/LuaPiGui.cpp

```diff
@@ -3740,7 +3734,6 @@ void LuaObject<PiGui::Instance>::RegisterClass()
 		{ "key_none", l_attr_key_none },
 		{ "key_shift", l_attr_key_shift },
 		{ "key_alt", l_attr_key_alt },
-		{ "keys", l_attr_keys },
```

Comment:
Q: why have the keys been moved to LuaInput.cpp, but the modifier keys not?

Reply:
The `pigui.keys` value is a static enumeration of SDLK_XXXX integer values. `pigui.key_alt`, `pigui.key_shift`, `pigui.key_ctrl` and `pigui.key_none` are instantaneous accessors which query the current ImGui keyboard state and return whether those specific keys are held down.

This is actually a separation of concerns as the previous `pigui.keys` had absolutely no relation to the `pigui.key_*` accessors.